scripts/travis/cleanup.js

```diff
@@ -1,23 +1,31 @@
-/*
- * Delete old artefacts from the testing db
+/**
+ * Delete old artefacts from the testing and staging dbs
  */
-const { UPLOAD_URL, BUILDS_SERVER, TRAVIS_BUILD_NUMBER } = process.env;
+const { UPLOAD_URL, BUILDS_SERVER, STAGING_SERVER, TRAVIS_BUILD_NUMBER } = process.env;
 const BUILDS_TO_KEEP = 50;
 const END_BUILD_TO_DELETE = TRAVIS_BUILD_NUMBER - BUILDS_TO_KEEP;
 const END_KEY = `medic:medic:test-${END_BUILD_TO_DELETE}`;
```

Comment:
END_KEY is now a pretty confusing name, since it's just for the testing server and not for the rest of them

Reply:
Inlined to where it's needed.